src/components/Listbox/components/TextOption/tests/TextOption.test.tsx

```diff
@@ -41,4 +42,19 @@ describe('TextOption', () => {
 
     expect(textOption).toContainReactComponent(Checkbox);
   });
+
+  it('does not render visual checkbox when allowMultiple is provided and isAction is true', () => {
+    const textOption = mountWithApp(
+      <ComboboxListboxOptionContext.Provider value={{allowMultiple: true}}>
+        <MappedActionContext.Provider value={{isAction: true}}>
+          <TextOption>child</TextOption>
+        </MappedActionContext.Provider>
+      </ComboboxListboxOptionContext.Provider>,
+      {
+        features: {newDesignLanguage: true},
```

Comment:
This feature was removed a long old time ago. You don't need this second argument here as it does nothing.

I guess you copied it from something up above; it can be removed there too